src/main/java/pt/lighthouselabs/obd/commands/fuel/FuelTrimObdCommand.java

```diff
@@ -46,27 +50,27 @@ private float prepareTempValue(final int value) {
 
   protected void performCalculations() {
     // ignore first two bytes [hh hh] of the response
-    fuelTrimValue = prepareTempValue(buffer.get(2));
+    percentage = prepareTempValue(buffer.get(2));
   }
 
-  @Override
-  public String getFormattedResult() {
-    return String.format("%.2f%s", fuelTrimValue, "%");
-  }
+    @Override
+    public String getFormattedResult() {
+        return String.format("%.2f%s", percentage, "%");
+    }
 
```

Comment:
seems to mix of 4 and 2 space indents , would you mind reformatting this bit to two?